### PR TITLE
Fix Failing Tests

### PR DESCRIPTION
**Pull request by lthall**

### CODE REVIEW

File: Tools/autotest/quadplane.py [hunk 1]
```diff
@@ -930,7 +930,6 @@ def tests(self):
 
         ret = super(AutoTestQuadPlane, self).tests()
         ret.extend([
-            self.AirMode,
```
Comment (peterbarker) [inline Tools/autotest/quadplane.py:933]: Can you provide a link to an instance of this failing, please?

File: Tools/autotest/arducopter.py [hunk 2]
```diff
@@ -9043,7 +9043,6 @@ def tests2b(self):  # this block currently around 9.5mins here
             self.RefindGPS,
             Test(self.GyroFFT, attempts=8),
             Test(self.GyroFFTHarmonic, attempts=8),
-            self.GyroFFTAverage,
```
Comment (peterbarker) [inline Tools/autotest/arducopter.py:9046]: This is better inserted in the `disabled_tests` method which is also in `arducopter.py`.  With a link to an issue saying why it was disabled....

**Review by peterbarker — CHANGES_REQUESTED** [review]
I can see removing GyroFFTAverage.  It's been flapping for a very long time.

I don't think I've ever seen that AirMode test on quadplanes failing.  That could be a regression.

File: Tools/autotest/quadplane.py (hunk 1 above)
Comment (lthall) [reply Tools/autotest/quadplane.py:933]: https://github.com/ArduPilot/ardupilot/tree/Copter-4.3

These are the two tests failing in Copter 4.3.

**Review by peterbarker — APPROVED** [review]

**Review by tridge — APPROVED** [review]

**Pull request merged**